app/views/staff/session_formats/create.js.erb

```diff
@@ -11,4 +11,5 @@
   $('#config-modal .errors').append('<%=  session_format.errors.full_messages.join(", ") %>');
```

Comment:
There's a minor issue with this append that came up during my last round of testing. If the form is submitted repeatedly, the previous errors don't get cleared out, but appended to the last errors that were displayed. Same with the other modals on that page. I think this'll be the last tweak and then we can merge this in.